tensorbay/opendataset/VGGFace2/__init__.py

```diff
@@ -0,0 +1,11 @@
+#!/usr/bin/env python3
```

Comment:
need to add this dataset to `docs/source/reference/api/opendataset.rst` and `tensorbay/opendataset/__init__.py`